pip-tools/dev-requirements.txt

```diff
@@ -100,7 +100,7 @@ marshmallow==3.13.0
     # via
     #   -r pip-tools/../requirements.txt
     #   environs
-mypy==0.910
+mypy
```

Comment:
Correct me if I'm wrong, but when you use pip-tools to manage dependencies, `dev-requirements.txt` or `requirements.txt` should always have a version number in them. However `dev-requirements.in` or `requirements.in` do not need to specify a version so if you do `piptools compile upgrade-package mypy` then it should update the version number in this file to the latest one

Reply:
yeah that was probably me just leaving in dev stuff, i.e. I had to test using no version to get it to work. I think it still works without the version, but it's more complete to have a version there, I'll change it now.